src/DatePicker/DatePicker.js

```diff
@@ -240,19 +243,22 @@ DatePicker.propTypes = {
     enableRangeSelection: PropTypes.bool,
     inputProps: PropTypes.object,
     locale: PropTypes.string,
+    value: PropTypes.string,
     onBlur: PropTypes.func
 };
 
 DatePicker.defaultProps = {
     locale: 'en',
-    onBlur: () => {}
+    onBlur: () => {},
+    value: ''
 };
 
 DatePicker.propDescriptions = {
     ...Calendar.propDescriptions,
     enableRangeSelection: 'Set to **true** to enable the selection of a date range (begin and end).',
     locale: 'Language code to set the locale.',
-    onBlur: 'Callback function for onBlur events. In the object returned, `date` is the date object and `formattedDate` is the formatted date.'
+    onBlur: 'Callback function for onBlur events. In the object returned, `date` is the date object and `formattedDate` is the formatted date.',
+    value: 'Default value to be shown in the Datepicker. The only accepted format is the ISO format, i.e. YYYY-MM-DD'
```

Comment:
Naming this something like `defaultValue` might be more clear, since it is only setting the initial `value` of the inner `<input>`, not forcing it to be static.